.github/workflows/pr.yaml

```diff
@@ -27,6 +27,8 @@ jobs:
   checks:
     secrets: inherit
     uses: rapidsai/shared-action-workflows/.github/workflows/checks.yaml@branch-23.04
+    with:
+      enable_check_generated_files: false
```

Comment:
I just tried to figure out what this flag does and from how I understand [this block](https://github.com/rapidsai/shared-action-workflows/blob/43490fb0808ace582f0f09c94e1e54be1e8b7e35/.github/workflows/checks.yaml#L68) it is fairly specific to the files generated by the rapids-dependency-file-checker. Can we maybe rename this flag so that this is a bit more obvious? From the name I would have guessed that it would skip all checks on all modifications applied by "checks" which I don't think would be desirable.

Reply:
I agree, it's not the clearest. However, I would like [to remove the dfg check entirely as a separate reusable action](https://github.com/rapidsai/shared-action-workflows/issues/46), so I don't think it's worth the effort of making further changes here since the flag only really needs to persist until every repo has moved to using the pre-commit hook.

Reply:
Sure.